AGENTS.md

```
@@ -0,0 +1,713 @@
+running `min check` until all addressed.
+
+
+
+## Troubleshooting FAQ
+
+### error: other: resolving dep '<package name>' by name: not found
+
+When using `min patched-pkg`, you can get an error if a package is not available locally, that looks
+like this:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)